examples/basic_example_v1/how-to-run.txt

```diff
@@ -0,0 +1,39 @@
+Running this sample with the managed YDB instance in Yandex Cloud
```

Comment:
I guess README.md is a better name for this file

Reply:
Converted to README.md.
Wanted to avoid Markdown syntax validation, by the way...